src/OpenAL/OpenToolkit.OpenAL/AudioContext.cs

```diff
@@ -399,13 +399,13 @@ IEnumerable<int> attributes
             if (!string.IsNullOrEmpty(device))
             {
                 _deviceName = device;
-                Device = (Device*)ALC.OpenDevice(device); // try to open device by name
+                Device = ALC.OpenDevice(device); // try to open device by name
```

Comment:
Those 2 comments here and line 408 can go, methinks.

Reply:
Yeah, I should really just remove this class though. It's some weird high level wrapper around AL contexts.

Reply:
This class is thrown in the trash in the latest